plutus-pab/src/Plutus/PAB/LocalCluster/Run.hs

```diff
@@ -179,6 +180,7 @@ runWith userContractHandler = withLocalClusterSetup $ \dir lo@LogOutputs{loClust
             void $ serveWallet
                 (NodeSource socketPath vData)
                 gp
+                tunedForMainnetPipeliningStrategy
```

Comment:
Should be configurable?

Reply:
Configurable with testnet as well?

Reply:
Well, we may provide it as on option. I was more talking about should we provide a CLI option to choose the strategy for local cluster users? Or they can just patch it themselves.